tests/integration/defs/disaggregated/sanity_check.sh

```diff
@@ -112,7 +112,7 @@ fi
 if [[ "${TEST_DESC}" != "gen_only" ]]; then
   expected_strings=("The capital of Germany is Berlin" "Asyncio is a Python library")
   if [[ "${TEST_DESC}" =~ "deepseek_v3_lite" ]]; then
-    expected_strings=("Berlin" "Asyncio is a powerful tool")
+    expected_strings=("Berlin" "Asyncio is a")
```

Comment:
Sometimes it prints is a library and sometimes it prints as tool so I shortened it.